modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1042,4 +980,46 @@ a|The query used a procedure that generated a warning.
 m|WARNING
 |Category
 m|GENERIC
-|===
\ No newline at end of file
+|===
+
+[#_neo_clientnotification_statement_RepeatedRelationshipReference]
+=== Neo.ClientNotification.Statement.RepeatedRelationshipReference
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
+|Title
+a|The query returns no results because a relationship variable is bound more than once.
+|Severity
+m|WARNING
+|Category
+m|GENERIC
+|===
+
+.Binding a relationship variable more than once.
+====
+Query::
++
+[source,cypher]
+----
+MATCH ()-[r]->()<-[r]-() RETURN count(*) AS count
+----
+
+Description of the returned code::
+A relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result.
```

Comment:
This should be the exact detail that they would get back in the description field of the same notification. So if we change it here, we also need to update the code. Since the cut is today, I believe it's better to keep it as it is in the code today:

```suggestion
A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
```